rti/mosaic-starter/src/main/java/org/eclipse/mosaic/starter/MosaicSimulation.java

```diff
@@ -376,7 +382,7 @@ private void initializeFederate(CRuntime.CFederate federate, FederateDescriptor
 
             if (StringUtils.isNotEmpty(federate.dockerImage)) {
                 descriptor.setFederateExecutor(
-                        descriptor.getAmbassador().createDockerFederateExecutor(federate.dockerImage, host.operatingSystem)
+                        descriptor.getAmbassador().createDockerFederateExecutor(federate.dockerImage, host.operatingSystem).setContainerName(simulationId)
```

Comment:
i would prefer to have the name of the federate name within the container name, e.g. `federate.id + "-" + simulationId`

Reply:
waiting for jenkins